vehicle/accel_brake_map_calibrator/accel_brake_map_calibrator/src/accel_brake_map_calibrator_node.cpp

```diff
@@ -791,6 +829,128 @@ void AccelBrakeMapCalibrator::executeUpdate(
                  .emplace_back(measured_acc);
 }
 
+bool AccelBrakeMapCalibrator::updateFourCellAroundOffset(
```

Comment:
There are so many if-condition for accel_mode in this function. 
Please define variables for the accel_mode at the top of the function, and remove the if-condition from the algorithm code, like below:

before
```
  const double zll = accel_mode ? update_accel_map_value_.at(accel_pedal_index).at(accel_vel_index)
                                : update_brake_map_value_.at(brake_pedal_index).at(brake_vel_index);
  const double zhl = accel_mode
                       ? update_accel_map_value_.at(accel_pedal_index + 1).at(accel_vel_index)
                       : update_brake_map_value_.at(brake_pedal_index + 1).at(brake_vel_index);
  const double zlh = accel_mode
                       ? update_accel_map_value_.at(accel_pedal_index).at(accel_vel_index + 1)
                       : update_brake_map_value_.at(brake_pedal_index).at(brake_vel_index + 1);
  const double zhh = accel_mode
                       ? update_accel_map_value_.at(accel_pedal_index + 1).at(accel_vel_index + 1)
                       : update_brake_map_value_.at(brake_pedal_index + 1).at(brake_vel_index + 1);
```

after
```
  auto & update_map_value = accel_mode ? update_accel_map_value_ : update_brake_map_value_;
  const auto & pedal_index = accel_mode ? accel_pedal_index : brake_pedal_index;
  const auto & vel_index = accel_mode ? accel_vel_index : brake_vel_index;

  const double zll = update_map_value.at(pedal_index).at(vel_index);
  const double zhl = update_map_value.at(pedal_index + 1).at(vel_index);
  const double zlh = update_map_value.at(pedal_index).at(vel_index + 1);
  const double zhh = update_map_value.at(pedal_index + 1).at(vel_index + 1);
```